docs/CDCDatabase-streamingsource.md

```diff
@@ -0,0 +1,30 @@
+# Change Tracking SQL Server Streaming Source
+
+Description
+-----------
+Streaming source for reading through Golden Gate Kafka topic.
```

Comment:
This plugin reads Change Data Capture (CDC) events from a Golden Gate Kafka topic.

Reply:
fixed